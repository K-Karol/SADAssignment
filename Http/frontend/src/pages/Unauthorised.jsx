import { Grid } from "@mui/material";
export default function Unauthorised() {
    return (
      <div>
        <Grid
          container
          spacing={{ xs: 2, md: 3 }}
          columns={{ xs: 4, sm: 8, md: 12 }}
        >
          <Grid item xs={4}>
            <h2>
                Sorry, you are not authorised to access this page. 
                If you believe this is a mistake, please contact your system administrator.
            </h2>
          </Grid>
        </Grid>
      </div>
    );
}