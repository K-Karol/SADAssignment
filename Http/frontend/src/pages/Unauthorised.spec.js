import React from 'react';
import renderer from 'react-test-renderer';
import Unauthorised from './Unauthorised';
 
 describe("GenerateCode page renders properly", () => {
    it("Matches DOM Snapshot", () => {
      const domTree = renderer.create(<Unauthorised />).toJSON();
      expect(domTree).toMatchSnapshot();
    });
  });