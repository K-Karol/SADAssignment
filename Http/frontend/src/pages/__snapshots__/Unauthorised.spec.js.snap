// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenerateCode page renders properly Matches DOM Snapshot 1`] = `
<div>
  <style
    dangerouslySetInnerHTML={
      Object {
        "__html": ".css-1xgjpwf-MuiGrid-root{box-sizing:border-box;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-flex-wrap:wrap;-webkit-flex-wrap:wrap;-ms-flex-wrap:wrap;flex-wrap:wrap;width:100%;-webkit-flex-direction:row;-ms-flex-direction:row;flex-direction:row;}@media (min-width:0px){.css-1xgjpwf-MuiGrid-root{margin-top:-16px;}.css-1xgjpwf-MuiGrid-root>.MuiGrid-item{padding-top:16px;}}@media (min-width:900px){.css-1xgjpwf-MuiGrid-root{margin-top:-24px;}.css-1xgjpwf-MuiGrid-root>.MuiGrid-item{padding-top:24px;}}@media (min-width:0px){.css-1xgjpwf-MuiGrid-root{width:calc(100% + 16px);margin-left:-16px;}.css-1xgjpwf-MuiGrid-root>.MuiGrid-item{padding-left:16px;}}@media (min-width:900px){.css-1xgjpwf-MuiGrid-root{width:calc(100% + 24px);margin-left:-24px;}.css-1xgjpwf-MuiGrid-root>.MuiGrid-item{padding-left:24px;}}",
      }
    }
    data-emotion="css 1xgjpwf-MuiGrid-root"
  />
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-spacing-md-3 css-1xgjpwf-MuiGrid-root"
  >
    <style
      dangerouslySetInnerHTML={
        Object {
          "__html": ".css-iltpxr-MuiGrid-root{box-sizing:border-box;margin:0;-webkit-flex-direction:row;-ms-flex-direction:row;flex-direction:row;-webkit-flex-basis:100%;-ms-flex-preferred-size:100%;flex-basis:100%;-webkit-box-flex:0;-webkit-flex-grow:0;-ms-flex-positive:0;flex-grow:0;max-width:100%;}@media (min-width:600px){.css-iltpxr-MuiGrid-root{-webkit-flex-basis:50%;-ms-flex-preferred-size:50%;flex-basis:50%;-webkit-box-flex:0;-webkit-flex-grow:0;-ms-flex-positive:0;flex-grow:0;max-width:50%;}}@media (min-width:900px){.css-iltpxr-MuiGrid-root{-webkit-flex-basis:33.333333%;-ms-flex-preferred-size:33.333333%;flex-basis:33.333333%;-webkit-box-flex:0;-webkit-flex-grow:0;-ms-flex-positive:0;flex-grow:0;max-width:33.333333%;}}",
        }
      }
      data-emotion="css iltpxr-MuiGrid-root"
    />
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-iltpxr-MuiGrid-root"
    >
      <h2>
        Sorry, you are not authorised to access this page. If you believe this is a mistake, please contact your system administrator.
      </h2>
    </div>
  </div>
</div>
`;
